Sources/_Datadog_TestRunner/DDTestRunner.swift

```diff
@@ -0,0 +1,138 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+@_implementationOnly import XCTest
+
+internal class DDTestRunner: NSObject, XCTestObservation {
+    static var instance: DDTestRunner?
+
+    let testNameRegex = try? NSRegularExpression(pattern: "([\\w]+) ([\\w]+)", options: .caseInsensitive)
+    let supportsSkipping = NSClassFromString("XCTSkippedTestContext") != nil
+    var currentBundleName = ""
+    var activeTestSpan: DDSpan?
+
+    override init() {
+        super.init()
+        XCTestObservationCenter.shared.addTestObserver(self)
+        startTracer()
+    }
+
+    func startTracer() {
+        var clientToken = ""
+        if let envToken = ProcessInfo.processInfo.environment["DATADOG_CLIENT_TOKEN"] {
+            clientToken = envToken.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
+        } else if let bundleToken = Bundle.main.infoDictionary?["DatadogClientToken"] as? String {
+            clientToken = bundleToken.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
+        }
+
+        if clientToken.isEmpty {
+            return
+        }
+
+        Datadog.initialize(
+            appContext: .init(),
+            configuration: Datadog.Configuration
+                .builderUsing(clientToken: clientToken, environment: CIEnvironmentValues.getEnvVariable("DD_ENV") ?? "test")
+                .set(serviceName: CIEnvironmentValues.getEnvVariable("DD_SERVICE") ?? ProcessInfo.processInfo.processName)
+                .build()
+        )
+
+        Global.sharedTracer = Tracer.initialize(
+            configuration: Tracer.Configuration(
+                sendNetworkInfo: true
+            )
+        )
+    }
+
+    func testBundleWillStart(_ testBundle: Bundle) {
+        currentBundleName = testBundle.bundleURL.deletingPathExtension().lastPathComponent
+    }
+
+    func testBundleDidFinish(_ testBundle: Bundle) {
+        guard let tracer = Global.sharedTracer as? DDTracer else {
+            return
+        }
+
+        /// We need to wait for all the traces to be written to the backend before exiting
+
+        tracer.queue.sync {} // waits until data is passed to writer
+        if let writer = TracingFeature.instance?.storage.writer as? FileWriter {
+            writer.queue.sync {} // waits until data is written
+        }
+        /// <--- here the file will be existing
+
+        do {
+            let directory = try Directory(withSubdirectoryPath: "com.datadoghq.traces/v1")
```

Comment:
The `"com.datadoghq.traces/v1"` path is shared with `TracingFeature` and it's important to keep it in sync. How about decoupling it to a `TracingFeature.dataDirectoryPath` constant?

Reply:
Added as constant